RELEASE.rst

```diff
@@ -1,6 +1,11 @@
 Release Notes
 =============
 
+Version 0.8.4
+-------------
+
+- Correct some imports
+
```

Comment:
Can you revert the change to this file? We have tooling that creates the release notes, and we're likely to roll this change up with copy/paste bugfix, #254

Reply:
Done